packages/terra-clinical-data-grid/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Added
+  * Defined roles for better screen reader experience
+  * New labelRef and descriptionRef props for allowing addition label and description of data-grid
```

Comment:
nit:
```suggestion
  * New `labelRef` and `descriptionRef` props for allowing addition label and description of data-grid
```

Also, did you mean, _additional_?

Reply:
Committed your suggestion and added this commit to have a better description of the changes https://github.com/cerner/terra-clinical/pull/832/commits/a7041d3c5f960636f0eec9355849a7a681e500b9